scripts/styleChecker.fsx

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env -S dotnet fsi
+
+#r "nuget: Fsdk, Version=0.6.0--date20230326-0544.git-5c4f55b"
+#load "../src/FileConventions/Helpers.fs"
+
+open System
+open System.IO
+
+open Fsdk
+open Fsdk.Process
+
+open Helpers
+
+let prettierVersion = "2.8.3"
+
+let StyleFSharpFiles() =
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments = "new tool-manifest --force"
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+    Process
+        .Execute(
+            {
+                Command = "dotnet"
+                Arguments =
+                    "tool install fantomless-tool --version 4.7.997-prerelease"
```

Comment:
@realmarv let's put this version at the top to make it easy to change, in the same way you did with the prettier version

Reply:
Done